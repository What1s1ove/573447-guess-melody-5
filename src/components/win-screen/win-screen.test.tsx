import * as React from 'react';
import { render } from 'enzyme';
import WinScreen from './win-screen';

describe(`WinScreen component`, () => {
  it(`should render WinScreen component`, () => {
    const component = render(<WinScreen />);

    expect(component).toMatchSnapshot();
  });
});
