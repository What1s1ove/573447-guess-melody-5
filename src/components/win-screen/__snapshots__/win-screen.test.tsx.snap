// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WinScreen component should render WinScreen component 1`] = `
<section
  class="result"
>
  <div
    class="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    class="result__title"
  >
    Вы настоящий меломан!
  </h2>
  <p
    class="result__total"
  >
    Вы ответили правильно на 0 вопросов и совершили 1 ошибки
  </p>
  <button
    class="replay"
    type="button"
  >
    Сыграть ещё раз
  </button>
</section>
`;
